.env.sample

```diff
@@ -1,13 +0,0 @@
-PORT=8080
```

Comment:
This file may need to be restored in this branch to maintain continuity on merging to main.